tests/Http/SavedSearches/UpdateSavedSearchRequestHandlerTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\SavedSearches;
+
+use Broadway\CommandHandling\Testing\TraceableCommandBus;
+use CultuurNet\UDB3\Deserializer\MissingValueException;
+use CultuurNet\UDB3\Http\ApiProblem\AssertApiProblemTrait;
+use CultuurNet\UDB3\Http\Request\Psr7RequestBuilder;
+use CultuurNet\UDB3\Http\Response\AssertJsonResponseTrait;
+use CultuurNet\UDB3\SavedSearches\Command\UpdateSavedSearch;
+use CultuurNet\UDB3\SavedSearches\Properties\QueryString;
+use Fig\Http\Message\StatusCodeInterface;
+use PHPUnit\Framework\TestCase;
+
+class UpdateSavedSearchRequestHandlerTest extends TestCase
```

Comment:
This misses a test for the unhappy path when a saved search does not exist.

Reply:
I tried to do this but the commandBus will not trigger _handleSubscribeToSavedSearch()_ . 
I already have a test for this unhappy path in tests/SavedSearches/UDB3SavedSearchesCommandHandlerTest.php:it_will_fail_to_update_a_nonexisting_saved_search .
Is that sufficent?